modules/order/src/Adjustment.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Drupal\commerce_order;
```

Comment:
That's what I was thinking, which is where the trait mentioned https://github.com/drupalcommerce/commerce/pull/447#discussion-diff-71970775 could go.